### DISCUSSION TITLE
MCMC guide for SS users

### DISCUSSION THREAD

**Discussion by k-doering-NOAA** [nmfs-stock-synthesis/stock-synthesis#426] [Team Posts]
Cole and I are working on putting together an MCMC guide for SS users based upon his adnuts work and workshop. Question: would this be better as a chapter/appendix to the SS manual, or a standalone guide like the [getting started with ss guide](https://github.com/nmfs-stock-synthesis/ss-documentation/blob/main/User_Guides/getting_started/Getting_Started_SS.pdf)?

**Comment by iantaylor-NOAA** [comment]
Unless it's really huge, I think it would be good to have it in the User Manual, similar to the essays at the end under "Guidance on Population Dynamics Modeling". It could also just replace the 2-page section on "Running MCMC" under "Running SS" (which would have to be updated regardless).

**Comment by chantelwetzel-noaa** [comment]
@iantaylor-NOAA  I had the same thought. I think there is a large subset of SS users that would be interested in exploring MCMC for their models and having in the manual would make that information easier to find.

**Comment by k-doering-NOAA** [comment]
Thanks for the feedback, Ian and Chantel! Sounds like I'll aim to keep it short and replace/augment the running MCMC section in the manual.
